catalog/glue/glue.go

```diff
@@ -202,16 +202,12 @@ func (c *Catalog) ListTables(ctx context.Context, namespace table.Identifier) it
 // LoadTable loads a table from the catalog table details.
 //
 // The identifier should contain the Glue database name, then Glue table name.
-func (c *Catalog) LoadTable(ctx context.Context, identifier table.Identifier, props iceberg.Properties) (*table.Table, error) {
+func (c *Catalog) LoadTable(ctx context.Context, identifier table.Identifier) (*table.Table, error) {
```

Comment:
the properties passed in allow overriding/adding properties to the returned table which may be necessary in certain circumstances